odl/__init__.py

```diff
@@ -41,6 +42,8 @@
 from . import phantom
 from . import deform
 from . import ufunc_ops
+from . import datasets
```

Comment:
How about having only one subpackage for data, containing both real data from wherever, and phantom data?